.gitignore

```diff
@@ -8,3 +8,5 @@ composer.phar
 
 # External Libraries
 /vendor/
+/index.php
+/npm-debug.log
```

Comment:
poprawna reguła to

/*.log

Nie chcemy aby żaden plik z logami, globalnie trafił do repozytorium.